ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -55,6 +55,46 @@ tests:
     - chain: openshift-qe-conc-builds
     workflow: openshift-qe-installer-aws
   timeout: 5h0m0s
+- as: sdn-to-ovn-24nodes
+  cron: 0 12 * * 6
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
+      OVN_SDN_MIGRATION_TIMEOUT: 3600s
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - ref: ovn-sdn-migration
+- as: sdn-to-ovn-120nodes
+  cron: 0 14 * * 0
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m6i.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.4xlarge
+      OVN_SDN_MIGRATION_TIMEOUT: 7200s
+      ZONES_COUNT: "3"
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - chain: ipi-install
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    - ref: ovn-sdn-migration
```

Comment:
The migration procedure here (https://github.com/openshift/release/blob/704d5ed102ed8e5d814736cee886fa5c4298d991/ci-operator/step-registry/ovn/sdn-migration/ovn-sdn-migration-commands.sh)  is offline migration that is relevant for < 4.15 clusters- since this job is a 4.15 job it should he using the online migration procedure.

Reply:
Switched to 4.14 to continue with offline migration.